stockroom/utils.py

```diff
@@ -18,14 +29,55 @@ def get_stock_root():
         path = path.parent
 
 
-def get_current_head(root: Path):
-    head = root.joinpath('head.stock')
-    with open(head, 'r') as f:
+def get_current_head(root: Path) -> str:
+    """
+    Reads the stock file and return the commit hash if found
+
+    :param root: The stock root path
+    :return: commit hash if found. Empty string other wises
+    """
+    with open(root/'head.stock', 'r') as f:
```

Comment:
Not how paths are handeled. use `pathlib.Path` class or `os.path.join` method.

Reply:
Linking to the previous answer: https://github.com/tensorwerk/stockroom/pull/1#discussion_r361903097

Reply:
Understood. no problem here :)